src/icons/CatalogIcon/CatalogIcon.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { DEFAULT_HEIGHT, DEFAULT_WIDTH } from '../../constants/constants';
+import { CARIBBEAN_GREEN, DARK_SLATE_GRAY, KEPPEL, WHITE, useTheme } from '../../theme';
+import { IconProps } from '../types';
+
+type CatalogIconProps = {
+  primaryFill?: string;
+  secondaryFill?: string;
+  tertiaryFill?: string;
+} & IconProps;
+
+export const CatalogIcon: FC<CatalogIconProps> = ({
+  width = DEFAULT_WIDTH,
+  height = DEFAULT_HEIGHT,
+  primaryFill,
+  secondaryFill,
+  tertiaryFill = WHITE,
+  style = {},
+  ...props
+}) => {
+  const theme = useTheme();
+  const themeMode = theme?.palette?.mode ?? 'light';
+
+  const themePrimaryFill = primaryFill ?? (themeMode === 'dark' ? KEPPEL : DARK_SLATE_GRAY);
+  const themeSecondaryFill = secondaryFill ?? (themeMode === 'dark' ? CARIBBEAN_GREEN : KEPPEL);
```

Comment:
@sudhanshutech @aabidsofi19 is this the correct way of using theme in Sistent? @dottharun you are able to test this right?

Reply:
Works in Meshery-UI when wrapped with <UsesSistent />.

Reply:
Ohh thankyou for confirming